content/docs/concepts/csidriver/features/powerstore.md

```diff
@@ -789,3 +805,23 @@ reclaimPolicy: Delete
 volumeBindingMode: WaitForFirstConsumer
 
 ```
+
+## Multi NAS Support
+The CSI PowerStore driver version 2.3.0 and later introduces multi-NAS support. This feature allows users to specify multiple NAS servers within a single storage class. By supporting multiple NAS servers in a single storage class, customers can create 2000 filesystems per PowerStore system. Previously, multiple storage classes were needed to support this configuration
```

Comment:
I think you need to update the driver version here, from 2.3.0 to 2.14.0

Reply:
per PowerStore system -> per PowerStore system across multiple NAS servers configured in SC?

Reply:
Updated